pages_unit_tests/establishment-address-select.test.js

```diff
@@ -3,8 +3,11 @@ import { mount, shallow } from "enzyme";
 
 const testCumulativeAnswers = {
   example: "test answer"
+ // editModeFirstPage: "registration-role"
```

Comment:
Remove comment

Reply:
Again no changes to this file anymore so believe this has already been done.